src/assets/tailwind.css

```diff
@@ -1,3 +1,11 @@
 @tailwind base;
 @tailwind components;
 @tailwind utilities;
+
```

Comment:
The tailwind.css file must contain just first 3 lines.
I you want to add general css for some components you should put it in a  base/common.scss file